File: JS/SimpleArraySum/SimpleArraySum.js
// DESCRIPTION
//Given an arr of ints, find the sum of eles.

// EXAMPLE
// ar = [1, 2, 3], 1 + 2 + 3 = 6

// OBJECTIVES
// Create a func called simpleArraySum
// Pass simpleArraySum arr
// Return sum of eles in arr

// CONSTRAINTS
// arr of ints between 0 and 1000 in val and quant

// SOLUTION
const simpleArraySum = (arr) => {
  let counter = null;
  for (let idx = 0; idx < arr.length; idx++) {
    counter += arr[idx];
  }
  return counter;
};

// TESTS
console.log(simpleArraySum([1, 2, 3, 4, 10, 11]));
// -> 31

File: JS/SimpleArraySum/SimpleArraySum.py
# transpile SimpleArraySum.js to Python

# given an array of integers find sum of integers

def simpleArraySum(arr):
    # variable to track total
    sum = 0
    # for loop to iterate array
    for integer in arr:
        sum += integer
    # each iteration adds to total tracker
    return sum


# TESTS
print(simpleArraySum([1, 2, 3, 4, 10, 11]))
# -> 31
